packages/fast-components-styles-msft/src/toggle/index.ts

```diff
@@ -0,0 +1,151 @@
+                    borderColor: (config: IDesignSystem): string => {
+                        return config.brandColor;
+                    },
+                    '&:hover': {
+                        backgroundColor: (config: IDesignSystem): string => {
+                            return hexToRGB(config.brandColor, .8);
+                        },
+                        borderColor: (config: IDesignSystem): string => {
+                            return hexToRGB(config.brandColor, .8);
+                        }
+                    },
+                    '&:focus': {
+                        borderColor: (config: IDesignSystem): string => {
+                            return config.brandColor;
+                        }
+                    },
+                    '& + span': {
+                        left: '28px',
+                        backgroundColor: (config: IDesignSystem): string => {
+                            return config.backgroundColor;
+                        }
+                    },
+                    '&:disabled': {
+                        cursor: 'not-allowed',
+                        background: (config: IDesignSystem): string => {
+                            return hexToRGB(config.foregroundColor, .2);
+                        },
+                        borderColor: 'transparent',
+                        '& + span': {
+                            background: (config: IDesignSystem): string => {
+                                return hexToRGB(config.foregroundColor, .2);
+                            }
+                        },
+                        '&:hover': {
+                            borderColor: 'transparent'
+                        }
+                    }
+                },
+                '&:not(:checked)': {
+                    background: 'transparent',
+                    borderColor: (config: IDesignSystem): string => {
+                        return config.foregroundColor;
+                    },
+                    '& + span': {
+                        backgroundColor: (config: IDesignSystem): string => {
+                            return config.foregroundColor;
+                        }
+                    },
+                    '&:disabled': {
+                        cursor: 'not-allowed',
+                        background: 'transparent',
+                        borderColor: (config: IDesignSystem): string => {
+                            return hexToRGB(config.foregroundColor, .2);
+                        },
+                        '& + span': {
+                            backgroundColor: (config: IDesignSystem): string => {
+                                return hexToRGB(config.foregroundColor, .2);
+                            }
+                        }
+                    }
+                },
+                '&:focus': {
+                    outline: '0'
+                }
+            }
+        }
+    }
+};
+
+export default styles;
```

Comment:
I think you need a newline here.

Reply:
We need our linter or whatever to yell about this.

Reply:
Yeah, it should be - not sure why it isn't

Reply:
I believe I fixed an issue in the linter with a PR, if it's not updating let me know and I'll see if I can repro on your branch. There was an issue with Mac not jumping into all files.

Reply:
Cool - will let you know.